packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -109,24 +100,6 @@ export default function PostOptionsMenu({
     shouldInvalidateQueries: false,
   });
 
-  const { toggleBookmark } = useBookmarkPost({
```

Comment:
Can we make this at the moment removed if in feed layout experiment just like we're doing for downvote?

Reply:
The bookmark is going to be regardless of feed layout. Either control or v1 will have the bookmark on card once it wins.

Reply:
I thought we were under the assumption that the bookmark on card wins? That was already what we had before.

Reply:
Ah, right. We only do that assumption on v1.